### PR TITLE
Migrate twig templates using single layout to JSX

### PR DESCRIPTION
**Pull request by lightnet328**
# WHAT

- Migrate twig templates using single layout
  - Search result page
  - Notification list page
  - API token page
  - Me page
  - Password page

# WHY

See #639

This PR depends on #642

### CODE REVIEW

File: lib/pages/Me/IndexPage.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      <div className="form-box">
+        <form action="/me" method="post" className="form-horizontal" role="form">
+          <fieldset>
+            <legend>{i18n.t('Basic Info')}</legend>
+            <div className="form-group row">
+              <label htmlFor="name" className="offset-1 col-sm-2 control-label">
+                {i18n.t('Name')}
+              </label>
+              <div className="col-sm-4">
+                <input id="name" className="form-control" type="text" name="userForm[name]" value={user.name} required />
```
Comment (saitoeku3) [inline lib/pages/Me/IndexPage.tsx:29]:
```suggestion
                <input id="name" className="form-control" type="text" name="userForm[name]" defaultValue={user.name} required />
```

File: lib/pages/Me/IndexPage.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+              </div>
+            </div>
+            <div className={classNames('form-group', 'row', user.email && 'has-error')}>
+              <label htmlFor="email" className="offset-1 col-sm-2 control-label">
+                {i18n.t('Email')}
+              </label>
+              <div className="col-sm-4">
+                <input id="email" className="form-control" type="email" name="userForm[email]" value={user.email || ''} required />
```
Comment (saitoeku3) [inline lib/pages/Me/IndexPage.tsx:37]:
```suggestion
                <input id="email" className="form-control" type="email" name="userForm[email]" defaultValue={user.email || ''} required />
```

File: lib/pages/Me/IndexPage.tsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                    value="en-US"
+                    checked={context.user.language === 'en-US'}
+                  />
+                  <label className="custom-control-label" htmlFor="lang-en">
+                    {i18n.t('English')}
+                  </label>
+                </div>
+                <div className="custom-control custom-radio custom-control-inline">
+                  {/* FIXME: Use constants */}
+                  <input type="radio" id="lang-ja" name="userForm[lang]" className="custom-control-input" value="ja" checked={context.user.language === 'ja'} />
```
Comment (saitoeku3) [inline lib/pages/Me/IndexPage.tsx:75]:
```suggestion
                  <input type="radio" id="lang-ja" name="userForm[lang]" className="custom-control-input" value="ja" defaultChecked={context.user.language === 'ja'} />
```

File: lib/pages/Me/IndexPage.tsx [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+              <div className="col-sm-4">
+                <div className="custom-control custom-radio custom-control-inline">
+                  {/* FIXME: Use constants */}
+                  <input
+                    type="radio"
+                    id="lang-en"
+                    name="userForm[lang]"
+                    className="custom-control-input"
+                    value="en-US"
+                    checked={context.user.language === 'en-US'}
```
Comment (saitoeku3) [inline lib/pages/Me/IndexPage.tsx:67]:
```suggestion
                    defaultChecked={context.user.language === 'en-US'}
```

File: lib/components/Alert.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+      ) : (
+        <ul>
+          {messages.map(message => (
+            <li key={message}>{message}</li>
+          ))}
+        </ul>
+      )}
+    </div>
+  ) : null
+}
```
Comment (saitoeku3) [inline lib/components/Alert.tsx:23]: An empty success alert is rendered now.
It seems that `<Alert />` should return `null` if messages is empty array.

Screen Shot 2020-04-27 at 23 48 29

Comment (lightnet328) [reply lib/components/Alert.tsx]: Thanks, I fixed it.

**Comment by saitoeku3** [comment]
@lightnet328  Please run `npm run format` before merging.

**Pull request merged**